src/lib/IbexaProductVersion.php

```diff
@@ -41,13 +42,23 @@ public static function getInstalledProduct(): string
         );
 
         // first element in the array is the package matching product edition
-        return reset($installedIbexaProducts);
+        $installedProduct = reset($installedIbexaProducts);
+
+        if (!is_string($installedProduct)) {
+            throw new RuntimeException(sprintf(
+                'Unable to establish installed version of Ibexa product. Packages ["%s"] does not contain any of ["%s"]. Check your installed packages.',
+                implode('","', $packages),
+                implode('","', self::IBEXA_PRODUCTS),
+            ));
```

Comment:
Not sure if hard exception is a good choice here for end-user. Or is this just for a Maintainer during install?

Reply:
Either this or calling effectively `trim(false, ',');`.

We could use some default instead.

Reply:
Well, actually, since the rest of the code relies on having an Ibexa product detected, it makes sense to throw an exception here (we haven't found an Ibexa product installed after all).